docker/redhat.Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal:7.8 AS builder
+
+# Update the builder packages and create user
+RUN microdnf update && rm -rf /var/cache/yum && \
```

Comment:
My OCD tendencies want to reformat this as:

```
RUN microdnf update \
 && rm -rf /var/cache/yum \
 && microdnf install shadow-utils \
 && microdnf clean all \
 && useradd -r -s /bin/false -U -G root cassandra
```

Reply:
That's entirely reasonable.